### DISCUSSION TITLE
Support MS Teams Notification channel

### DISCUSSION THREAD

**Discussion by calebdw** [laravel/framework#47124] [Ideas] [answered]
Hello!

I noticed that Laravel supports Slack as a notification channel and even integrates with Laravel Horizion. However, it would also be nice to send notifications to MS Teams.

Thanks!

**Comment by henzeb** [comment]
https://github.com/laravel-notification-channels/microsoft-teams

**Reply by calebdw** [reply]
Cool beans! Is it possible to hook Horizon up to it?

**Reply by henzeb** [reply, marked as answer]
sadly not with a magic method with routeSlackNotificationsTo. But you can listen to the JobFailed event. https://laravel.com/docs/10.x/queues#failed-job-events
